contracts/cw3-fixed-multisig/src/contract.rs

```diff
@@ -213,10 +214,14 @@ pub fn execute_close(
     let mut prop = PROPOSALS.load(deps.storage, proposal_id)?;
     if [Status::Executed, Status::Rejected, Status::Passed]
         .iter()
-        .any(|x| *x == prop.status)
+        .any(|&s| s == prop.status)
```

Comment:
Nitpicking, but it looks like:
```rust
if  [Status::Executed, Status::Rejected, Status::Passed].contains(&prop.status)
```

Reply:
Ignore me - just realized you changed it this way in follow-up.

Reply:
This is a valid suggestion. Applied it in both multisigs.